src/account_mt5.py

```diff
@@ -26,6 +27,11 @@ def get_positions(self) -> tuple: # Note on 1.21.24 that MT5 returns positions a
         if positions is None:
             raise RuntimeError('No positions returned from MT5. Error is ' + str(mt5.last_error() or ''))
         return positions
+    
+    def get_deal_history(self) -> dict:
```

Comment:
you have this method here but also inside the 'handlers.py' file and currently it looks to be getting called from 'handler.py'. If we intend to use it we should be retrieving the data from here.